testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -56,6 +56,66 @@ public static void DeleteBuild()
             }
         }
 
+        [MenuItem(MultiprocessBaseMenuName + "/Windows Standalone Player")]
+        public static void BuildWindowsStandalonePlayer()
```

Comment:
not a huge fan of duplicating build settings in separate methods like this. With this, if I want to change the test scene path for example, I need to change it in 3 locations. Same for buildPlayerOptions.

Reply:
I tried to clean this up @SamuelBellomo , please take a look at the updated code.

Reply:
Any reason BuildRelease and BuildDebug don't use your code too?

Reply:
I didn't feel comfortable making changes to your original code and intentions but I will if you are okay with it. @SamuelBellomo

Reply:
Did you test locally from the editor? If that works, then you didn't mess anything up :)